### PR TITLE
VACMS-8166: Upgrade to Drupal 9.4.5

### LINKED ISSUE #8166: Upgrade to Drupal 9.4
## Description
We should upgrade to Drupal 9.4 when it is released (presumably June).

## Notes
- The 9.3 upgrade ended up changing almost 900 files, causing substantial issues with merge conflicts on Sitewide PRs
- The 9.3 upgrade did not price in a substantial amount of work; we might want to turn this into a discovery ticket and assume that we might need to open up a followup ticket to adequately estimate the lift for this process.

## Acceptance Criteria
- [ ] Drupal 9.4.x is deployed, works.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

Closes #8166.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CODE REVIEW

File: docroot/modules/custom/va_gov_notifications/va_gov_notifications.install [hunk 1]
```diff
@@ -9,7 +9,7 @@
  * Ininstall messages stack.
  */
 function va_gov_notifications_update_9001(&$sandbox) {
-  module_load_install('va_gov_db');
+  \Drupal::moduleHandler()->loadInclude('va_gov_db', 'install');
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_notifications/va_gov_notifications.install:12]: Nice fix.  for long past fured hook updates we could always remove them if they needed something crazy to keep them current.  ;)  I do like keeping them for historical context though if the lift (like this one) is not too big.

File: config/sync/views.view.search.yml [hunk 2]
```diff
@@ -245,7 +245,7 @@ display:
               vba_facility: default
               vet_center: default
       query:
-        type: views_query
+        type: search_api_query
```
Comment (swirtSJW) [inline config/sync/views.view.search.yml:248]: interesting

**Review by swirtSJW — COMMENTED** [review]
Given the changes, the only thing I see at risk is to test that knowledge base search still works.

**Comment by ndouglas** [comment]
@swirtSJW I checked the [knowledge base search](https://cms-dp7azpjt7riinmyw7wnx1zepifwxrpzc.ci.cms.va.gov/admin/content/audit/knowledge-base-article-administration) and couldn't find any way in which it differed from prod appearance or behavior.

Not interested in merging it tonight anyway 🙂

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**